### ISSUE TITLE
[iOS] Can't run sample project on simulator.

### ISSUE THREAD

**Issue by mirroringcontact** [ant-media/Ant-Media-Server#4973]
### Short description 
Your sample project is working on real device, but i can't run it on simulator.  I see error: https://shottr.cc/s/EOnK/SCR-20230405-8kd.png

### Environment 
Xcode 14.3, iPhone 14 Pro simulator

### Steps to reproduce
1. Clone sample app https://github.com/ant-media/WebRTC-iOS-SDK
2. Run on simulator

**Comment by mekya** [comment]
Hi @mirroringcontact ,

Thank you for the issue. What's your device model and CPU?

**Comment by mirroringcontact** [comment]
Intel, macbook pro 2017

**Comment by mekya** [comment]
Yes, it's a known issue and it's not working on Intel devices. The simulator is working in M1 and M2 chips.
